mico-admin/src/app/service-detail/service-detail.component.html

```diff
@@ -41,9 +41,10 @@ <h3>{{service?.name}}</h3>
             <mico-service-dependency-graph [shortName]="shortName" [version]="selectedVersion">
             </mico-service-dependency-graph>
         </mat-tab>
-        <!-- DEPLOYMENT TAB-->
-        <mat-tab label="Deployment">
-            Service-Detail-Deployment...
+        <!-- KUBECONFIG TAB-->
+        <mat-tab label="kube config yaml">
```

Comment:
Choose a different label. First letter should be in uppercase.
Proposal: Kubernetes Configuration